### PR TITLE
Update module1-exercises.tex

### PR DESCRIPTION
**Pull request by qrn99**
Added 3 more questions (one conceptual, two challenging as I ran out of ideas). Solutions will be typed up later if these questions are accepted.

Sorry for the hassle of branch naming problems :(

### CODE REVIEW

File: problemsets-mat223/modules/module1-exercises.tex [hunk 1]
```diff
@@ -57,7 +57,7 @@
 					$\mat{5\\-5}$ and
 					$\Set*{\vec v\in\R^2 \given
 					\vec v=k\mat{1\\-1}\text{ for some }k\in\R}$
-				\item $\mat{-1 \\ 2 \\ 1}$ and
+				\item $\mat{-1 \\ 2 \\ 1}$ and \\
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:60]: Why did you force linebreaks? This is generally a bad idea...

File: problemsets-mat223/modules/module1-exercises.tex [hunk 2]
```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:131]: -> Use set-builder notation to describe the following sets

File: problemsets-mat223/modules/module1-exercises.tex [hunk 3]
```diff
@@ -126,6 +126,52 @@
...
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:133]: We use the word `coordinate` since `component` is associated with vector component. You need to spell it out a bit more. These are first years. E.g. "The set of vectors in R^2 whose coordinates are rational numbers."

File: problemsets-mat223/modules/module1-exercises.tex [hunk 4]
```diff
@@ -126,6 +126,52 @@
...
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:135]: This is not good...Maybe, "The set of vectors consisting only of [1,2]"

File: problemsets-mat223/modules/module1-exercises.tex [hunk 5]
```diff
@@ -126,6 +126,52 @@
...
+		        \item $\R^3$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:136]: What are you hoping for an answer for this one?

File: problemsets-mat223/modules/module1-exercises.tex [hunk 6]
```diff
@@ -126,6 +126,52 @@
...
+		    \end{enumerate}
+    		
 		% Challenge (3 questions)
+		\prob 
+		% Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your answers.
+		    \begin{enumerate}
+		        \item The y-axis in $\R^3$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:145]: Name the set so you can explicitly refer to it in the options. I would suggest $\mathcal Y$.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 7]
```diff
@@ -126,6 +126,52 @@
...
+		        \begin{enumerate}
+		            \item It can be written as a set of linear combinations of vectors.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cup B$.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cap B$.
+		            \item For any vectors $\vec v \in \emptyset$, $\vec v \neq \vec 0$
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:150]: What does this have to do with the set? It should be a separate question. Use `For all` instead of `For any`.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 8]
```diff
@@ -126,6 +126,52 @@
...
+		            \item For some vectors $\vec v \in \emptyset$, $\vec v \neq \vec 0$.
+		        \end{enumerate}
+		        \item $\emptyset$
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:153]: I don't want to over-emphasize about the empty set. Perhaps the set of vectors whose coordinates are $\pm 3$?

File: problemsets-mat223/modules/module1-exercises.tex [hunk 9]
```diff
@@ -126,6 +126,52 @@
...
+		\prob 
+		% Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your answers.
+		    \begin{enumerate}
+		        \item The y-axis in $\R^3$.
+		        \begin{enumerate}
+		            \item It can be written as a set of linear combinations of vectors.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cup B$.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cap B$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:149]: These are always true. Can you come up with something that's not always true? Otherwise, it belongs as part of a question: "Determine whether the following statements are Always True, Sometimes True, or Never True"

File: problemsets-mat223/modules/module1-exercises.tex (hunk 1 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:60]: The original overlaps with added questions as the text for these are too wide. I tried to use align but it didn't turn out well. Is there a better way to avoid overlapping text?

Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:60]: Take out the forced newlines. I will deal with formatting when edits are complete.

File: problemsets-mat223/modules/module1-exercises.tex (hunk 4 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:135]: My intention was to distinguish a collection of sets from a set of vectors, an object confusion mistake that students had in the past. Is this too indirect?

File: problemsets-mat223/modules/module1-exercises.tex (hunk 5 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:136]: I was hoping that students will consider linearly independent/basis vectors and use linear combinations to express this set. I agree this is too general and there are many acceptable answers...

File: problemsets-mat223/modules/module1-exercises.tex (hunk 7 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:150]: Make Line 148-149 to a separate question: use set operations to describe some sets?

Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:150]: Oh sorry this are typos. It should be vectors in the set of $\mathcal Y$ which is the y-axis.

File: problemsets-mat223/modules/module1-exercises.tex (hunk 8 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:153]: I used the empty set to distinguish from the previous set y-axis for the last two statements, as "for all x, P(x)" implies "for some x, P(x)" in general, but it is not the case for the empty set. It is a small detail about quantifiers. I got a feeling from marking last midterm's definition questions, some (or many) students do not understand quantifiers well. I can think about something better that doesn't use the empty set, or I can move these to the next question that asks for proof/counterexample?

File: problemsets-mat223/modules/module1-exercises.tex (hunk 4 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:135]: Yes. Empty sets are harder than non-empty sets. You could do something like, "Let $P(\vec x) = {\vec x, -\vec x}". The set ${P(e_1), P(e_2), P(e_3)$

File: problemsets-mat223/modules/module1-exercises.tex (hunk 5 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:136]: Yeah. Let's take it away.

File: problemsets-mat223/modules/module1-exercises.tex (hunk 8 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:153]: Go ahead and change it to my suggestion.

**Comment by siefkenj** [comment]
I see you've marked a lot of conversations resolved. Do you have an update to push, or are you still working on it?

**Comment by qrn99** [comment]
> I see you've marked a lot of conversations resolved. Do you have an update to push, or are you still working on it?

Still working on the last comment.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 10]
```diff
@@ -126,6 +126,84 @@
...
+		answers.
+		\begin{enumerate}
+			\item $\mathcal{Y}$, the y-axis in $\R^{3}$.
+				\begin{enumerate}
+					\item $\mathcal{Y}$ is a finite set.
+
+					\item Let
+						$\mathcal{A} = \{\vec a \in \R^{3} : \vec a = \beta \vec v
+						\text{ for some }\vec v \in \mathcal{Y}, \beta \in \R \}$, then $\mathcal{A}
+						\subseteq \mathcal{Y}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:154]: Use `\Set{... \given ...}` to do set builder stuff.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 11]
```diff
@@ -126,6 +126,84 @@
...
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, $\vec v \neq \vec 0$。
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:156]: Use `we have` after the comma so it doesn't look  like a bunch of math mushed together.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 12]
```diff
@@ -126,6 +126,84 @@
...
+
+					\item For some vectors $\vec v \in \mathcal{Y}$, $\vec v \neq \vec 0$.
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, there exists a vector
+						$\vec x \in \mathcal{Y}$ such that $\vec x + \vec v = \vec e_{2}$.
+
+					\item There exists a vector $\vec x \in \mathcal{Y}$ such that for
+						all vectors $\vec v \in \mathcal{Y}$, $\vec x + \vec v = \vec e
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:164]: `we have` again.

File: problemsets-mat223/modules/module1-exercises.tex [hunk 13]
```diff
@@ -126,6 +126,84 @@
...
+					\item For all vectors $\vec v \in \mathcal{S}$, there exists a vector
+						$\vec x \in \mathcal{S}$ such that $\vec x + \vec v = \vec 0$.
+
+					\item There exists a vector $\vec x \in \mathcal{S}$ such that for
+						all vectors $\vec v \in \mathcal{S}$, $\vec x + \vec v = \vec 0$.
+				\end{enumerate}
+		\end{enumerate}
+
+		\prob % Propose: Give students some practices on proving general statements about sets
+		Determine whether the following statement is correct or not. If it is, prove it.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:191]: => "for each of the following statements, determine ..."

File: problemsets-mat223/modules/module1-exercises.tex [hunk 14]
```diff
@@ -126,6 +126,84 @@
...
+		Otherwise, give a counterexample.
+		\begin{enumerate}
+			\item For all sets $A \subseteq \R^{n}$, $A$ can be written as a set of linear
+				combinations of vectors in $\R^{n}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:195]: I don't know what this means...

File: problemsets-mat223/modules/module1-exercises.tex [hunk 15]
```diff
@@ -126,6 +126,84 @@
...
+
+			\item If $A \subseteq B$, then $A \cap B = A$.
+
+			\item If $A \subseteq B$, then $A \cap B \neq B$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:199]: Include the reverse of these inclusions as well.

**Comment by qrn99** [comment]
> This is getting close! Go ahead and add your name to the `contributors.tex` file as well! (If you want to be attributed).

Thats some good news before my class starts today! I think I am already attributed :)

File: problemsets-mat223/modules/module1-exercises.tex (hunk 14 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:195]: I mean to ask: Can any set of vectors be written as a set of linear combination of vectors.

File: problemsets-mat223/modules/module1-exercises.tex (hunk 15 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:199]: As making two new statements like: `If $B \subseteq A$, then $A \cap B = A$.` and `If $B \subseteq A$, then $A \cap B \neq B$.` or put the reverse inclusions into the original statements?

File: problemsets-mat223/modules/module1-exercises.tex (hunk 14 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:195]: So, would the answer "yes, because v=1v for all vectors. Therefore `X= { v : v = 1x for some x\in X}`" be what you are looking for?

File: problemsets-mat223/modules/module1-exercises.tex (hunk 15 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module1-exercises.tex:199]: As in additional statements.

File: problemsets-mat223/modules/module1-exercises.tex (hunk 14 above)
Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:195]: I agree that this isn't very intellectual stimulating... Will remove it!

**Comment by qrn99** [comment]
**Still require formatting for overlapping texts.**

File: problemsets-mat223/modules/module1-exercises.tex [hunk 16]
```diff
@@ -126,6 +126,90 @@
...
+		
+		\prob Use set-builder notation to describe the following sets.
+		\begin{enumerate}
+			\item The set of vectors in $\R^{2}$ whose coordinates are rational numbers.
+
+			\item The set of vectors in $\R^{2}$ whose coordinates are irrational
+				numbers.
+
+			\item Let $P(\vec x) = \{\vec x, -\vec x\}$. The set $\{P(\vec e_{1}), P(
+				\vec e_{2})\}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:138]: Use `\Set{...}`

File: problemsets-mat223/modules/module1-exercises.tex [hunk 17]
```diff
@@ -126,6 +126,90 @@
...
+		\end{enumerate}
+
 		% Challenge (3 questions)
+		\prob % Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your
+		answers.
+		\begin{enumerate}
+			\item $\mathcal{Y}$, the y-axis in $\R^{3}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:147]: -> $y$-axis

File: problemsets-mat223/modules/module1-exercises.tex [hunk 18]
```diff
@@ -126,6 +126,90 @@
...
+				\begin{enumerate}
+					\item $\mathcal{Y}$ is a finite set.
+
+					\item Let
+						$\mathcal{A} = \Set*
+						{\vec a \in \R^3 \given \vec a = \beta \vec v \text{ for some } \vec v \in \mathcal{Y}, \beta \in \R}$,
+						then $\mathcal{A} \subseteq \mathcal{Y}$.
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, we have $\vec v \neq
+						\vec 0$。
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1-exercises.tex:157]: Is that a Japanese period?

Comment (qrn99) [reply problemsets-mat223/modules/module1-exercises.tex:157]: Ops, accidentally typed in Chinese keyboard...I think a Japanese period is the same as a Chinese period.
